unstract/connectors/src/unstract/connectors/databases/oracle-db/oracle_db.py

```diff
@@ -0,0 +1,165 @@
+import datetime
+import os
+from typing import Any
+
+import oracledb
+from oracledb.connection import Connection
+
+from unstract.connectors.databases.unstract_db import UnstractDB
+
+
+class OracleDB(UnstractDB):
+    def __init__(self, settings: dict[str, Any]):
+        super().__init__("OracleDB")
+
+        self.config_dir = settings["config_dir"]
+        self.user = settings["user"]
+        self.password = settings["password"]
+        self.dsn = settings["dsn"]
+        self.wallet_location = settings["wallet_location"]
+        self.wallet_password = settings["wallet_password"]
```

Comment:
I don;t know how it actually works in other connectors. But this apporach like `dict[value]` have a chance of getting keyError at the root. I think it would be better if we change it to `dict.get(valye)` then `None` check.

If possible make the changes now, Or make it later in another PR

Reply:
done